WordPress/Classes/Services/ReaderCardService.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+class ReaderCardService {
+    private let service: ReaderPostServiceRemote
+    private let coreDataStack: CoreDataStack
+
+    init(service: ReaderPostServiceRemote = ReaderPostServiceRemote(wordPressComRestApi: WordPressComMockrestApi()),
+         coreDataStack: CoreDataStack = ContextManager.shared) {
+        self.service = service
+        self.coreDataStack = coreDataStack
+    }
+
+    func fetch(success: @escaping (Int, Bool) -> Void, failure: @escaping (Error?) -> Void) {
+        service.fetchCards(for: ["foo", "bar"],
```

Comment:
I'm not sure yet how this will be in the final implementation so I haven't added the argument.

It makes sense that `fetch` would accept a list of interests but right now this list of interests will come from the Reader Stream VC, something that definitely doesn't make sense. I would prefer to fetch all user's interests here than in the VC.

Given that I don't know yet how this will be I just left the placeholder there.

Reply:
That makes sense, thanks for the explanation.